.github/workflows/ci.yml

```diff
@@ -77,7 +77,26 @@ jobs:
           git config user.name "CI"
       - run: |
           sudo apt-get update
-          sudo apt-get install clang mold ccache build-essential cmake ninja-build pkgconf python3-zmq libevent-dev libboost-dev libsqlite3-dev systemtap-sdt-dev libzmq3-dev qt6-base-dev qt6-tools-dev qt6-l10n-tools libqrencode-dev -y
+          sudo apt-get install -y \
+            build-essential \
+            capnproto \
+            ccache \
+            clang \
+            cmake \
+            libcapnp-dev \
+            libevent-dev \
+            libboost-dev \
```

Comment:
In commit "cmake: enable ENABLE_IPC option by default" (969a9472a43b7e8c0ae1b8d0dd1a71e5d118ea0c)

Maybe sort these while you are splitting them up. libboost-dev libqrencode-dev and systemtap-sdt-dev seem to be the only entries out of order.

Reply:
I did sort them, but missed a few.

Reply:
Fixed the sort